backend/app/api/internal/api.py

```diff
@@ -12,6 +14,184 @@
+    #: String to use for ENSEMBL version.
+    ensembl: str
+    #: Version of dbNSFP.
+    dbnsfp: str
+    #: Version of dbscSNV.
+    dbscsnv: str
+    #: Version of CADD.
+    cadd: str
+    #: Version of gnomAD for constraints.
+    gnomad_constraints: str
+    #: Version of gnomAD mtDNA.
+    gnomad_mtdna: str
+    #: Version of gnomAD v2.
+    gnomad_v2: str
+    #: Version of gnomAD v3.
+    gnomad_v3: str
+    #: Version of gnomAD v4.
+    gnomad_v4: str
+    #: Version of gnomAD SV.
+    gnomad_sv: str
+    #: Version of gnomAD CNV v4.
+    gnomad_cnv4: str
+    #: Version of gnomAD SV v4.
+    gnomad_sv4: str
+    #: Version of dbVar.
+    dbvar: str
+    #: Version of DGV.
+    dgv: str
+    #: Version of DGV Gold Standard.
+    dgv_gs: str
+    #: ExAC CNVs.
+    exac_cnv: str
+    #: Thousand Genomes SVs.
+    g1k_svs: str
+    #: HelixMtDb
+    helixmtdb: str
+    #: UCSC conservation (GRCh37).
+    ucsc_cons_37: str
+    #: UCSC conservation (GRCh38).
+    ucsc_cons_38: str
+    #: UCSC repeat masker (GRCh37).
+    ucsc_rmsk_37: str
+    #: UCSC repeat masker (GRCh38).
+    ucsc_rmsk_38: str
+    #: UCSC genomicSuperDups (GRCh37).
+    ucsc_genomic_super_dups_37: str
+    #: UCSC genomicSuperDups (GRCh38).
+    ucsc_genomic_super_dups_38: str
+    #: UCSC genome browser altSeqLiftOverPsl (GRCh37).
+    ucsc_alt_seq_liftover_37: str
+    #: UCSC genome browser altSeqLiftOverPsl (GRCh38).
+    ucsc_alt_seq_liftover_38: str
+    #: UCSC genome browser fixSeqLiftOverPsl (GRCh37).
+    ucsc_fix_seq_liftover_37: str
+    #: UCSC genome browser fixSeqLiftOverPsl (GRCh38).
+    ucsc_fix_seq_liftover_38: str
+    #: RefSeq version (GRCh37).
+    refseq_37: str
+    #: RefSeq version (GRCh38).
+    refseq_38: str
+    #: dbSNP version.
+    dbsnp: str
+    #: ACMG secondary findings version.
+    acmg_sf: str
+    #: HPO
+    hpo: str
+    #: Orphadata
+    orphadata: str
+    #: Pathogenic MMS
+    patho_mms: str
+    #: Mehari transcript data.
+    mehari_tx: str
+    #: ClinVar release.
+    clinvar_release: str
+    #: ClinVar version.
+    clinvar_version: str
+    #: RefSeq functional elements for GRCh37.
+    refseq_fe_37: str
+    #: RefSeq functional elements for GRCh38.
+    refseq_fe_38: str
+
+
+#: The current date.
+TODAY = datetime.date.strftime(datetime.date.today(), "%Y%m%d")
+
+#: The data versions to use.
+DATA_VERSIONS = DataVersions(
+    annonars="0.2.1",
+    viguno="0.36.1",
+    mehari="0.25.4",
+    cada_prio="0.6.1",
+    dotty="0.4.1",
+    variant_validator="2.2.1",
+    intervar="2021-08",
+    autocnv="latest",
+    pubtator3="latest",
+    beacon_network="latest",
+    alphamissense="1",
+    clingen_gene=TODAY,
+    clingen_variant=TODAY,
+    ensembl_37="87",
+    ensembl_38="109",
+    ensembl="111",
+    dbnsfp="4.5",
+    dbscsnv="1.1",
+    cadd="1.6",
+    gnomad_constraints="4.0",
+    gnomad_mtdna="3.1",
+    gnomad_v2="2.1.1",
+    gnomad_v3="3.1.2",
+    gnomad_v4="4.0",
+    gnomad_sv="2.1.1",
```

Comment:
svs exist both for v2 and v4 at least when looking on their download page: https://gnomad.broadinstitute.org/downloads